pages/[id].jsx

```diff
@@ -1,20 +1,57 @@
+import { useEffect, useState } from 'react';
 import { useRouter } from 'next/router';
 import Head from 'next/head';
 
+import apiService from '../services/apiService';
+import LoadingStatus, { loadingStates } from '../components/LoadingStatus';
+
 export default function DetailPage() {
+  const [teamMemberDetails, setTeamMemberDetails] = useState({});
+  const [loadingStatus, setLoadingStatus] = useState('EMPTY');
   const router = useRouter();
   const { id } = router.query;
 
+  useEffect(() => {
+    const getAndSetData = async () => {
+      try {
+        setLoadingStatus('LOADING');
+        const fetchedTeamMember = await apiService.getTeamMemberById('api/fellowship/', id);
+        setTeamMemberDetails(fetchedTeamMember);
+        setLoadingStatus('LOADED');
+      } catch (err) {
+        setLoadingStatus('ERROR');
+      }
+    };
+
+    getAndSetData();
+  }, [id]);
+
   return (
     <div>
       <Head>
-        <title>{`${id} | Sparkbox Team Availability`}</title>
-        <meta name="description" content={`View details about ${id}, including their projects, skills, and interests.`} />
+        <title>
+          {teamMemberDetails && teamMemberDetails.firstName}
+          {' '}
+          | Sparkbox Team Availability
+        </title>
+        <meta name="description" content={`View details about ${teamMemberDetails && teamMemberDetails.firstName}, including their projects, skills, and interests.`} />
       </Head>
       <main>
-        <h1>
-          {`Hello ${id}!`}
-        </h1>
+
+        <LoadingStatus
+          loadingStatus={loadingStates[loadingStatus]}
```

Comment:
With the other `loadingState` changes, you can just pass in `loadingStatus` here.